tests/generic_config_updater/test_pfcwd_status.py

```diff
@@ -228,7 +228,7 @@ def test_start_pfcwd(duthost, extract_pfcwd_config, ensure_dut_readiness, stop_p
     try:
         tmpfile = generate_tmpfile(duthost)
         output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
-        if is_valid_platform_and_version(duthost, "PFC_WD", "PFCWD enable/disable"):
+        if is_valid_platform_and_version(duthost, "PFC_WD", "PFCWD enable/disable", "add"):
```

Comment:
>add [](http://example.com/codeflow?start=85&length=3)

If this `add` depends on above `op`, better to use variable and do not repeat constant strings.

Reply:
This test case only tests 'add', I moved add to be in variable `op` which would help if we extend the test case